### DISCUSSION TITLE
Output path

### DISCUSSION THREAD

**Discussion by foxyzboi** [streamlink/streamlink#4477] [Q&A] [answered]
How can I change the output path?

**Comment by mkbloke** [comment, marked as answer]
That's rather vague.  Do you mean when using `-o` / `--output`?

Just set the output as you would usually, but with an absolute or relative path prepended to the output file name.

**Reply by foxyzboi** [reply]
> That's rather vague. Do you mean when using `-o` / `--output`?
> 
> Just set the output as you would usually, but with an absolute or relative path prepended to the output file name.

Yes, I mean `-o`
I write `streamlink <URL> best -o <filename>`. By default video saves in C:\Users\Admin. So I want to change save path.

**Reply by mkbloke** [reply]
So, set it to `C:\Users\AnotherUserName\FileName.ext` or `C:\Users\Admin\Some\Other\Path\FileName.ext`?

**Reply by foxyzboi** [reply]
> So, set it to `C:\Users\AnotherUserName\FileName.ext` or `C:\Users\Admin\Some\Other\Path\FileName.ext`?

I would like to change the save path to a different drive

**Reply by mkbloke** [reply]
Change the drive letter then?
